docs/Configuration.md

```diff
@@ -0,0 +1,119 @@
+# Configuring Just Saying
+
+## The publish subscribe pattern
+> In software architecture, publish–subscribe is a messaging pattern where senders of messages, called publishers, do not program the messages to be sent directly to specific receivers, called subscribers, but instead categorize published messages into classes without knowledge of which subscribers, if any, there may be. Similarly, subscribers express interest in one or more classes and only receive messages that are of interest, without knowledge of which publishers, if any, there are. [(wikipedia)](https://en.wikipedia.org/wiki/Publish%E2%80%93subscribe_pattern)
+
+# Configuring a single publisher and a single subscriber
+
+## Configuring the publisher
+
+- In this configuration the following AWS resources will be created
+    - a SNS topic of name `orderplacedevent`
+    - a SQS queue of name `orderplacedevent`
+    - a SNS topic subscription for queue `orderplacedevent` to topic `orderplacedevent`
+
+- In `startup.cs` add the following
+```cs
+    services.AddJustSaying(config =>
+    {
+        config.Publications(x =>
```

Comment:
In these examples I'd be tempted to use more verbose naming than `x`.

Reply:
Not sure what to call the builders - what do you think?
```cs
    services.AddJustSaying(configureBus =>
    {
        configureBus.Subscriptions(configureSubscription =>
        {
            // Creates the SQS queue
            // Creates the SQS to SNS subscription
            configureSubscription.ForTopic<OrderPlacedEvent>();
        });
    }
```

Reply:
Maybe `bus` and then `subscription`, so it's like the below so the code lines read a bit like written English?

```csharp
    services.AddJustSaying(bus =>
    {
        bus.Subscriptions(subscription =>
        {
            // Creates the SQS queue
            // Creates the SQS to SNS subscription
            subscription.ForTopic<OrderPlacedEvent>();
        });
    }
```

Reply:
will do - thats better - mine was too long

Reply:
done